pkgs/development/haskell-modules/configuration-nix.nix

```diff
@@ -439,6 +439,7 @@ self: super: builtins.intersectAttrs super {
     dontCheck
     enableSeparateBinOutput
     (self.generateOptparseApplicativeCompletions [ "postgrest" ])
+    (overrideCabal { passthru.tests = pkgs.nixosTests.postgrest; })
```

Comment:
Another noob question. This means that when building nixpkgs, it should prefer running the nixos test in this PR instead of the postgREST test suite right?